### PR TITLE
Enable `semi: true`

### PR DESCRIPTION
**Pull request by simonswine**
Speaking with @grafakus he suggested we should enable semicolons, to make code more readable. Not too sure if that is something controversial or not.

Requires #82 to be merged first

### CODE REVIEW

File: commitlint.config.js [hunk 1]
```diff
@@ -1 +0,0 @@
-module.exports = {extends: ['@commitlint/config-conventional']}
```
Comment (grafakus) [inline commitlint.config.js:1]: Why?

Comment (simonswine) [reply commitlint.config.js:1]: Removed it as part of https://github.com/grafana/pyroscope-nodejs/pull/76 (but not fully)

File: src/profilers/wall-profiler.ts [hunk 2]
```diff
@@ -1,118 +1,118 @@
...
       ...oldLabels,
       ...lbls,
-    })
-    ;(fn as AnyWrappedCallback)(...args)
+    });
+    (fn as AnyWrappedCallback)(...args);
     this.setLabels({
       ...oldLabels,
-    })
+    });
```
Comment (simonswine) [inline src/profilers/wall-profiler.ts:69]: This is made a lot more readable by `semi=true`

**Review by bryanhuhta — APPROVED** [review]
LGTM

**Pull request merged**